test/clj/rems/test/api/applications.clj

```diff
@@ -911,8 +911,8 @@
                                          :application-id application-id
                                          :items {}})
                              app)]
-          (is (not= 200 (:status try-submit)))
-          (is (= "unauthorized" (read-body try-submit)))))
+          (is (= 400 (:status try-submit)))
```

Comment:
Hmm should this be 200 OK but some `:success false` and errors by the logic of our recent discussions?

Reply:
I think this is different, you're actually violating an API invariant (old-style request for new-style application)